runtime-modules/proposals/engine/src/lib.rs

```diff
@@ -589,13 +590,35 @@ impl<T: Trait> Module<T> {
 
     /// Resets voting results for active proposals.
     /// Possible application includes new council elections.
-    pub fn reset_active_proposals() {
+    pub fn reset_votes_for_active_proposals() {
```

Comment:
Why are `reset_votes_for_active_proposals` and `reactivate_pending_constitutionality_proposals` two distinct API calls? Wouldn't you just want a single call which you invoke whenever you want dormant proposals to be reactivated?

Reply:
Because they are different:
- reset_votes_for_active_proposals - is a call to reset proposals votes on council elections. To clear obsolete votes for a new council.
- reactivate_pending_constitutionality_proposals - makes active again proposals with constitutionality

Even if I introduce a new combined method I would keep both `reactivate_pending_constitutionality_proposals` and `reset_votes_for_active_proposals` for clarity.